tests/Aql/CalculationExecutorTest.cpp

```diff
@@ -236,6 +237,126 @@ TEST_F(CalculationExecutorTest, there_are_rows_in_the_upstream_the_producer_wait
   ASSERT_FALSE(row.produced());
 }
 
+TEST_F(CalculationExecutorTest, test_produce_datarange) {
+  // This fetcher will not be called!
```

Comment:
Isn't it easier to use the ExecutorTestHelper?

Tests are ok though

Reply:
This will be part of the C++ Refactor later on (After sub and end will be merged).